src/icons/Nft.tsx

```diff
@@ -0,0 +1,38 @@
+import CommonIconsProps from 'icons/CommonIconsProps'
+import classnames, { stroke, strokeWidth } from 'classnames/tailwind'
+
+export default function ({ inheritStrokeColor }: CommonIconsProps) {
+  const pathStyles = classnames(
+    strokeWidth('stroke-2'),
+    stroke(inheritStrokeColor ? 'stroke-inherit' : 'stroke-formal-accent')
+  )
+
+  return (
+    <svg
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M3 15.5L4.29288 8.62621C4.60509 6.96628 6.97375 6.94155 7.32055 8.5946V8.5946C7.64121 10.1231 9.76908 10.2632 10.2873 8.7899L12.5 2.5"
+        className={pathStyles}
+        stroke-linecap="round"
```

Comment:
Hm, maybe we should place it inside the `index.css` as a default value?
```css
path, svg, circle {
 stroke-linecap: round
}
```
So we can remove it from everywhere